src/gui_common/CreditsScroll.cs

```diff
@@ -605,7 +605,12 @@ private string GetTranslatedHeading(string team)
 
         foreach (var columnText in splitTexts)
         {
-            var label = new CustomExpandingWordWrappedLabel(columnText.ToString());
+            var label = new Label
+            {
+                Text = columnText.ToString(),
+                HorizontalAlignment = HorizontalAlignment.Center,
+                SizeFlagsHorizontal = SizeFlags.ExpandFill,
```

Comment:
Shouldn't this actually specify smart word wrap? (and minimum width of 50 pixels)

Reply:
I ask because I recall that Labels don't have word wrap enabled by default... so this *shouldn't* work correctly without word wrap enabled.

Reply:
Seems smart word wrap throws the text much lower than where it should be. Regular word (not smart) doesn't do that though.

Reply:
Smart is needed because if someone has a name like "thisisareallylongwordwithnospacesinitthatpreventsnormalwordwrapfromworkingatallonthisname" things will break.

Having the labels be contained in a hbox is probably the best way to ensure they can expand and take up all of the horizontal space to make things work well. Though ensuring uniform column widths might be tiny bit complicated.

Reply:
Well, that `label` is already added as child to the hbox so I don't know why that's happening.

Reply:
Make sure the hbox is also expand and the minsize is set to equal to screen width.